auth-web/src/stores/staff.ts

```diff
@@ -251,16 +251,21 @@ export const useStaffStore = defineStore('staff', () => {
   }
 
   async function searchReviews (filterParams: ReviewFilterParams) {
-    const response = await BusinessService.searchReviews(filterParams)
-    if (response?.data) {
-      return {
-        limit: response.data.limit,
-        page: response.data.page,
-        total: response.data.total,
-        reviews: response.data.reviews
+    try {
+      const response = await BusinessService.searchReviews(filterParams)
+      if (response?.data) {
+        return {
+          limit: response.data.limit,
+          page: response.data.page,
+          total: response.data.total,
+          reviews: response.data.reviews
+        }
```

Comment:
Why return a new object here? Can't you just return `response.data`?

Reply:
In fact, why not move this block into BusinessService.searchReviews(), so that it returns an object of type ReviewList (instead of the Axios response)?

I think it's better to keep the Axios stuff contained in the services file. THIS file shouldn't know about an Axios response.

Reply:
I used the similar structure as line 240 function searchOrgs() in this file. If we move this block into BusinessService.searchReviews(), it looks like we don't need the function in the staff.ts file anymore? We can call BusinessService.searchReviews() directly in the vue.

Reply:
Maybe...

Normally a services method should throw an exception if there's a network error (403, 404, 5xx, etc) and the caller should handle that. However, I'm OK if you decide to handle the error in the services method and return null or `[]` accordingly, so that you can use the service method directly in the Vue template.

Reply:
Moved the function in staff.ts to BusinessService.searchReviews(). Now call BusinessService.searchReviews() directly in table.vue